blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,211 @@
+
+**Impact:** We could immediately respond to the customer and confirm the behavior as well as provide more context for the estimated time to fix.
+
+**Time Saved:** 18 hours waiting for a teammate to answer my question—hours that were especially important given we were actively working with a customer.
+
+### What's the Sourcegraph standard for error message syntax?
+
+I needed to finish writing the error messages that display to users for a major new feature. We have [error message guidelines](https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#error-messages) in our handbook, but it’s easier to quickly learn something with an example.
+
+**Search:** [This search](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+error+try&patternType=regexp) quickly gave me dozens of in-product examples.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTgx?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 300px;"></iframe>
+</p>
+
+**Impact:** I was able to produce more unified product messaging.
+
+**Time Saved:** I saved 10 minutes I would have spent clicking through the product and avoided causing errors I would have otherwise needed to cause so I could find live examples.
+
+### Situation: Determine the different [blank]@sourcegraph emails we direct users to across our entire user-facing codebase
+
+I needed to add a contact email to some new documentation, but wasn’t sure what email addresses we’d already set up and which would be the best to reuse.
+
+**Search:** [This search](https://sourcegraph.com/search?q=context:global+r:sourcegraph/+mailto:...%40sourcegraph.com+-file:test+count:all&patternType=structural) showed me what email addresses were already in use.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTgy?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 300px;"></iframe>
+</p>
+
+**Impact:** We learned we have a “feedback@sourcegraph” email that was better than the “support@” email I was about to use.
+
+**Time Saved:** We saved hours or even days. The alternative solution would have involved searching every single repository or asking every long-tenured teammate to list what emails they recall seeing used.
+
+**Follow-up situation:** What anchor text do we generally use for the email link, or do we display the direct email? [Search](https://sourcegraph.com/search?q=context:global+r:sourcegraph/+mailto:feedback%40sourcegraph.com&patternType=literal):
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTg0?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 300px;"></iframe>
+</p>
+
+### Should we use “prototype” or “experimental” for early-stage product features?
```

Comment:
@nickmyyz so this one depends on seeing the search count, but counts don't display in notebooks. Can we add to the body the current counts?